src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/JenkinsBuildPlanCreator.java

```diff
@@ -82,7 +82,7 @@ public JenkinsBuildPlanCreator(ResourceLoaderService resourceLoaderService) {
 
     @PostConstruct
     public void init() {
-        this.artemisNotificationUrl = ARTEMIS_SERVER_URL + "/api" + Constants.NEW_RESULT_RESOURCE_PATH;
+        this.artemisNotificationUrl = ARTEMIS_SERVER_URL + "/api/" + Constants.NEW_RESULT_RESOURCE_PATH;
```

Comment:
```suggestion
        this.artemisNotificationUrl = ARTEMIS_SERVER_URL + Constants.NEW_RESULT_RESOURCE_API_PATH;
```

There seems to already exist a constant that contains exactly what is wanted here. https://github.com/ls1intum/Artemis/blob/1abf87528a2bb7188f0c920d20f3e4b69f9c298d/src/main/java/de/tum/in/www1/artemis/config/Constants.java#L42

Reply:
Thanks for the suggestion!